packages/docusaurus-theme-bootstrap/src/theme/Navbar/index.tsx

```diff
@@ -63,9 +63,10 @@ function NavItem({
 }
 
 function Navbar() {
+  console.log(useDocusaurusContext());
   const {
     siteConfig: {
-      themeConfig: {navbar: {title, links = []} = {}},
+      themeConfig: {navbar: {title = '', items: links = []} = {}},
```

Comment:
not urgent right now but we'd rather add themeConfig validation/normalization to bootstrap theme too to make it production ready and avoid putting default values in comps everywhere